queue_services/entity-auth/src/entity_auth/services/gcp_queue.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, app: Flask = None):
         self._publisher = None
 
         if app:
-            self.app_init(app)
+            self.init_app(app)
 
     def init_app(self, app: Flask):
```

Comment:
Line 65 was throwing a lint error as there is no `app_init` on the `GCPQueue` class. Can somebody confirm whether this was meant to be `self.init_app`?